### ISSUE TITLE
"splitBrain" alert is being triggered without reason

### ISSUE THREAD

**Issue by ebenzecri** [scylladb/scylla-monitoring#2254]
*Installation details*
Panel Name: N/A
Dashboard Name: N/A
Scylla-Monitoring Version: 4.7.0
Scylla-Version: Multiple versions

*Issue*
```
  - alert: splitBrain
    expr: sum(scylla_gossip_live) >= (count(scylla_node_operation_mode==3)-1) * count(scylla_gossip_live)
    for: 10m
    labels:
      severity: "warn"
      description: 'Cluster in a split-brain mode'
      summary: Some nodes the cluster do not see all of the other live nodes
```
is being triggered without a real reason and probably because `expr` is wrong. Also `summary` must be rewritten.

*Proof*
![](https://github.com/scylladb/scylla-monitoring/assets/62569533/fed1b69b-86ec-4aec-8114-aa750f0b2f50)

Number of nodes: 3
Live nodes that each node can see: 2
`nodetool status` executed on each node doesn't show any node that has a state different to `UN`

**Expressions results**
`sum(scylla_gossip_live)` = 6
`count(scylla_node_operation_mode==3)-1` = 2
`count(scylla_gossip_live)` = 3

**Comment by mykaul** [comment]
> Also `summary` must be rewritten.

rewritten how?

**Comment by ebenzecri** [comment]
> > Also `summary` must be rewritten.
> 
> rewritten how?

I'm not a big fan of suggesting how things should be written, but a suggestion can be:
```
Not all nodes of the cluster are seeing the same number of live nodes
```

Anything that improves the current text will be OK.

**Comment by ebenzecri** [comment]
@amnonh Could you please create a new release or include this in 4.7.0? Thanks

**Comment by amnonh** [comment]
It's already backported to 4.7 branch and I'll create a patch release soon, I re-released 4.7.0
